package.json

```
@@ -5,12 +5,12 @@
   "main": "index.js",
   "scripts": {
     "build": "tsc",
-    "start": "node index.js"
+    "start": "node build/index.js"
```

To not require users to run both `build` and `start` we could 

```suggestion
    "start": "npm build && node build/index.js"
```

Although that doesn't work on Windows, which lacks `&&` support. Another alternative would be adding `ts-node` to the package so building is not required anymore:

```suggestion
    "start": "ts-node index.ts"
```

Can also stay as it is, I just wanted to make some suggestions.

---

I like that I will go with the ts-node as more compatibility is never a bad thing.